packages/lib/survey/service.ts

```diff
@@ -965,7 +967,9 @@ export const getSyncSurveys = async (
         const personActionClassIds = Array.from(
           new Set(personActions?.map((action) => action.actionClass?.id ?? ""))
         );
-        const personUserId = person.userId ?? person.attributes?.userId ?? "";
+
+        const attributes = await getAttributes(person.id);
+        const personUserId = person.userId ?? "";
```

Comment:
I think an empty string isn't a good choice here. Can we also set the `personUserId` to undefined if not set?